netci.groovy

```diff
@@ -1574,10 +1574,10 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
 
                     def buildArchitecture = 'arm'
 
-                    // For 'arm' (the RyuJIT/arm32 architecture), tell build.cmd to use RyuJIT/arm32 for crossgen compilation.
-                    // RyuJIT/arm32 is currently not the default JIT; it is an aljit. So, this is a special case.
+                    // For 'armlb' (the RyuJIT/arm32 architecture), tell build.cmd to use RyuJIT/arm32 for crossgen compilation.
```

Comment:
Nit. misleading comments.
Please update comment something like ` // For 'armlb' (the legacy backend for arm), tell build.cmd to use legacy backend for crossgen compilation.`

Reply:
Fixed